plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeRegisterTableProcedureUsingGlueMetastore.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake;
+
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.hive.metastore.HiveMetastore;
+import io.trino.plugin.hive.metastore.glue.DefaultGlueColumnStatisticsProviderFactory;
+import io.trino.plugin.hive.metastore.glue.GlueHiveMetastore;
+import io.trino.plugin.hive.metastore.glue.GlueHiveMetastoreConfig;
+
+import java.util.Map;
+import java.util.Optional;
+
+import static com.google.common.util.concurrent.MoreExecutors.directExecutor;
+import static io.trino.plugin.hive.HiveTestUtils.HDFS_ENVIRONMENT;
+
+public class TestDeltaLakeRegisterTableProcedureUsingGlueMetastore
+        extends BaseDeltaLakeRegisterTableProcedureTest
+{
+    @Override
+    protected HiveMetastore createTestMetastore()
+    {
+        return new GlueHiveMetastore(
+                HDFS_ENVIRONMENT,
+                new GlueHiveMetastoreConfig(),
+                DefaultAWSCredentialsProviderChain.getInstance(),
+                directExecutor(),
+                new DefaultGlueColumnStatisticsProviderFactory(directExecutor(), directExecutor()),
+                Optional.empty(),
+                table -> true);
+    }
+
+    @Override
+    protected Map<String, String> getConnectionProperties()
+    {
+        return ImmutableMap.of(
+                "delta.unique-table-location", "true",
+                "hive.metastore", "glue",
+                "hive.metastore.glue.region", "us-east-2",
```

Comment:
Remove the line. The value comes over `AWS_REGION` environment variable.

https://github.com/trinodb/trino/blob/d4707c49c935f04177df669dc4a592d7c45d2546/.github/workflows/ci.yml#L594